spec/spec_helper.rb

```diff
@@ -0,0 +1,99 @@
+require 'coveralls'
+require 'rspec'
+require 'pry-rails'
+require 'shoulda-matchers'
+require 'launchy'
+Coveralls.wear!('rails')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.